style.css

```diff
@@ -504,15 +506,18 @@ input {
 .form {
   flex: 1;
   margin-top: 36.5px;
-  margin-left: auto;
-  margin-right: auto;
-  width: 327px;
+  width: 80%;
+}
+
+input[type='text'] {
+  margin-bottom: -10px;
```

Comment:
## [OPTIONAL]

- [ ] You can also improve the styling when the input is focused,  but removing the outline on it. just like
 
```
input:focus {
  outline: none
}
```